chain/epoch-manager/src/adapter.rs

```diff
@@ -523,6 +530,19 @@ impl EpochManagerAdapter for EpochManagerHandle {
         Ok(ShardUId::from_shard_id_and_layout(shard_id, &shard_layout))
     }
 
+    fn shard_ids_to_uids(
```

Comment:
as mentioned in an earlier comment, let's try to not add this function to epoch_manager

Reply:
Ok, I got rid of it.